plugins/portfolio/controllers/blog.js

```diff
@@ -394,7 +394,7 @@ module.exports = function BlogModule(pb) {
         if(this.req.pencilblue_article || this.req.pencilblue_page) {
             cb(null, content.headline + ' | ' + pb.config.siteName);
         }
-        else if(searchId = this.req.pencilblue_section || this.req.pencilblue_topic) {
+        else if(searchId = this.req.pencilblue_section || this.req.pencilblue_topic) { // Is this an assignment during evaluation?
```

Comment:
yes, the order of operations coalesces the section and topic before assigning them to the search id variable